.github/native-tests.json

```diff
@@ -39,7 +39,7 @@
         {
             "category": "Data6",
             "timeout": 50,
-            "test-modules": "elasticsearch-rest-client, elasticsearch-rest-high-level-client, hibernate-search-orm-elasticsearch, hibernate-reactive-panache",
+            "test-modules": "elasticsearch-rest-client, elasticsearch-rest-high-level-client, hibernate-search-orm-elasticsearch, hibernate-search-orm-elasticsearch-tenancy, hibernate-search-orm-opensearch, hibernate-reactive-panache",
```

Comment:
You don't need a higher timeout, there?

Reply:
I don't know. The two modules I added are not large, but I suppose native compilation takes a long time even for small projects.

I'll raise the timeout to 65, we'll see.